django/db/models/expressions.py

```diff
@@ -507,6 +507,15 @@ def as_sql(self, compiler, connection, function=None, template=None):
         template = template or self.extra.get('template', self.template)
         return template % self.extra, params
 
+    def as_sqlite(self, *args, **kwargs):
+        sql, params = self.as_sql(*args, **kwargs)
+        try:
```

Comment:
@MarkusH: Because that's apparently how one accesses the output_field property: if the output field can not be resolved, the property throws an exception instead of just returning None. The alternative would be to use the private method _output_field_or_none()